### PR TITLE
Add Keynote Speaker Arch Robison

### PR DESCRIPTION
**Pull request by Bubblemelon**
Related issue: [#33](https://github.com/JuliaCon/JuliaCon2019/issues/33). :eyes: See issue thread for screenshot.

### CODE REVIEW

**Comment by xorJane** [comment]
LGTM! The 1st photo (less round) seems better. The 2nd one looks distorted.

@ninjin

**Comment by cormullion** [comment]
Should it be "principal" rather than "principle"? And "parallel" perhaps?

**Comment by Bubblemelon** [comment]
@cormullion thanks for catching those! :smile:

**Comment by ninjin** [comment]
It should be “SIMD”, not “SMID”. Otherwise it looks all good to go.

File: 2019/content19.html [hunk 1]
```diff
@@ -80,27 +80,47 @@ <h2>Keynote Speakers</h2>
...
+          name="Arch Robison"
+          url="http://www.blonzonics.us/home"
+          avatar="/2019/assets/img/archrobison.jpg"
+          affiliation="NVIDIA"
+          abstract="Arch D. Robison is a Principal Systems Software Engineer at NVIDIA, where he works 
+          on [TensorRT](https://developer.nvidia.com/tensorrt), NVIDIA's platform for high-performance
+          deep-learning inference. He was the lead developer for KAI C++, the original architect of Intel
+          Threading Building Blocks, and one of the authors of the book *Structured Parallel Programming:
+          Patterns for Efficient Computation*. Arch contributed type-based alias analysis and vectorization
+          support to Julia, including the original implementation of [SIMD in Julia 0.3](https://software.
```
Comment (Bubblemelon) [inline 2019/content19.html]: "SMID" is now "SIMD"

**Comment by ninjin** [comment]
Thanks a ton @Bubblemelon! Let’s get this show on the road!

**Pull request merged**